test/simulator/index.js

```diff
@@ -7,7 +7,7 @@ function loadTests(desc, dir) {
 	let contents = fs.readdirSync(path.join(__dirname, dir));
 	describe(desc, function () {
 		for (let i = 0; i < contents.length; i++) {
-			if (contents[i].substr(-3) === '.js') require('./' + path.join(dir, contents[i]));
+			if (contents[i].substr(-3) === '.js') require('./' + path.join(dir, contents[i].substr(0, contents[i].length - 3)));
```

Comment:
I think so. Assume we have the same resolution pattern described above and `contents = ['hello.js', 'hello.es6']`. It will skip `hello.es6`, and run `require('./<div>/hello')` and actually load `hello.es6`.

That being said, I don't know how often people will want to override the tests. It would also require the user to customize the mocha options. The use for this is drastically lower than the non-test code.

If you want me to leave this line alone, just say the magic words.